ietf/doc/views_doc.py

```diff
@@ -1103,6 +1106,8 @@ def document_bibtex(request, name, rev=None):
             if rev == h.rev:
                 doc = h
                 break
+        else:
+            raise Http404()
```

Comment:
This is a good place to return an error, but I think this view should also outright reject an rfc document with any rev (as you did for `document_main()`)

Reply:
Addressed